src/store/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, applyMiddleware } from 'redux'
+import thunkMiddleware from 'redux-thunk'
+import update from '../reducers/datapackageReducer'
+
+export default function configureStore(preloadedState) {
+  return createStore(
+    update,
```

Comment:
I think root reducer will be useful. Right now we have only one reducer. If in future we have multiple reducers we can use combine reducer to pass to store. e.g.
```javascript
//reducer/index.js

import { combineReducers } from 'redux';
import { routerReducer } from 'react-router-redux';
import update from "./datapackageReducer";

const rootReducer = combineReducers({
	routing: routerReducer,
	update: update,
});

export default rootReducer;
```
Then in store/configureStore.js
```javaScript
createStore(rootReducer, initialState, compose(
		applyMiddleware(...middlewares)
		)
```